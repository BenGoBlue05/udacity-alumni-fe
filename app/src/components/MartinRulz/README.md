## MartinRulz Component
A component that ...

### Example

```js
<MartinRulz />
```

### Props

| Prop          | Type     | Default     | Possible Values
| ------------- | -------- | ----------- | ---------------------------------------------
| **myProp**    | String   |             | Any string value


### Other Information
