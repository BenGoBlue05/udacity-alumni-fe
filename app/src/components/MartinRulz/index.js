import React from 'react';
import styles from './index.module.scss';
import cssModules from 'react-css-modules';
import Box from 'grommet-udacity/components/Box';
import Heading from 'grommet-udacity/components/Heading';
import Button from 'grommet-udacity/components/Button';
import Section from 'grommet-udacity/components/Section';

const MartinRulz = () => (
  <Section>
    <Box
      pad="large"
      align="center"
      justify="center"
    >
      <Heading tag="h2" align="center">Under Construction</Heading>
      <img
        className={styles.image}
        src="http://oi63.tinypic.com/24vse9f.jpg"
      />
    </Box>
    <Box
      align="center"
    >
      <Heading tag="h3">
        If you feel like helping us build this page
      </Heading>
      <Button href="mailto:forbiddenvoid@gmail.com" label="Get in Touch!" />
    </Box>
  </Section>
);

export default cssModules(MartinRulz, styles);
