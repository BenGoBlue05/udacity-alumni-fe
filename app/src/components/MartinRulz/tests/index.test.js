import MartinRulz from '../index';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import React from 'react';

describe('<MartinRulz />', () => {
  it('should render with default props', () => {
    const wrapper = shallow(
      <MartinRulz />
    );
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
