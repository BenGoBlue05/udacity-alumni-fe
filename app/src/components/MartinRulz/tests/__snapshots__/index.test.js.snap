exports[`<MartinRulz /> should render with default props 1`] = `
<Section
  pad={
    Object {
      "vertical": "medium"
    }
  }>
  <Box
    align="center"
    announce={false}
    direction="column"
    focusable={true}
    justify="center"
    pad="large"
    responsive={true}
    tag="div">
    <Heading
      align="center"
      tag="h2">
      Under Construction
    </Heading>
    <img
      src="http://oi63.tinypic.com/24vse9f.jpg" />
  </Box>
  <Box
    align="center"
    announce={false}
    direction="column"
    focusable={true}
    pad="none"
    responsive={true}
    tag="div">
    <Heading
      tag="h3">
      If you feel like helping us build this page
    </Heading>
    <Button
      href="mailto:forbiddenvoid@gmail.com"
      label="Get in Touch!"
      type="button" />
  </Box>
</Section>
`;
